README.md

```diff
@@ -8,152 +8,225 @@
+system (e.g., environmental variables, home folder, shared memory, important
+locations in the root filesystem).
+
+Bear in might that Toolbox is not meant to be "safe" in terms of isolating your
+actions in the container from the host. It strives to extend your workflow
+rather than isolate it using the container technology.
+
+**Uh.., what?**
+
+Let's say you want to try to use `ansible` but you don't want to bloat your
+system (e.g., Fedora Workstation) or you can't really install it comfortably
+(e.g., Silverblue).
+
+Just create a container with `toolbox create`, enter it with `toolbox enter`
+(you should see the '⬢' symbol at the beginning of the command prompt) and
+type `sudo dnf install ansible` (the container is Fedora-based).
+
+When the installation is done you should have `ansible` prepared for use,
+without affecting the base operating system (if you leave the container,
+`ansible` will no longer be available until you return into it).
+
+And because your home folder is automatically mounted in the container, you
+can easily execute your already existing ansible playbooks.
+
+When you're done, you can simply exit the container by typing `exit` or
+pressing `Ctrl+D` and delete the container with `toolbox rm` and the system
+should be in the same state when you started (apart from files you may have
+created/edited/removed in your home folder or other locations shared with the
+host's filesystem).
+
+Also notice that you did not need to run Toolbox with `sudo` or as a
+priviledged user!
+
+**Sounds good! So, to use Toolbox I need one of those mentioned OSs?**
+
+No, you don't! This tool **doesn't require** using an OSTree-based system — it
+works equally well if you're running e.g. existing Fedora Workstation or
+Server. But if you haven't given Silverblue a try then you definitely should!
 
-Toolbox requires at least Podman 1.4.0 to work, and uses the Meson build
-system.
+## Documentation
 
-The following dependencies are required to build it:
-- meson
-- go-md2man
+https://docs.fedoraproject.org/en-US/fedora-silverblue/toolbox/
+
+## Installation
+
+### Fedora Silverblue
+
+Toolbox is installed by default on Fedora Silverblue.
+
+### Fedora
+
+`$ dnf install toolbox`
+
+[Package definition](https://src.fedoraproject.org/rpms/toolbox)
+
+### Arch Linux
+
+`$ pacman -S toolbox`
+
+[Package definition](https://www.archlinux.org/packages/community/x86_64/toolbox/)
+
+## How to build from source
+
+Toolbox uses [Meson](https://mesonbuild.com) buildsystem for building,
+installing and generating manual pages + bash completion.
+
+> Because `Meson` does [not](https://github.com/mesonbuild/meson/issues/123) have native support for Go, Toolbox uses a wrapper
+> script ([`src/go-build-wrapper`](https://github.com/containers/toolbox/blob/master/src/go-build-wrapper))
+> to execute the build.
+
+**The following dependencies are required to build Toolbox:**
+
+- podman (>=v1.4.0)
+- flatpak-session-helper (monitoring of key system files)
```

Comment:
Strictly speaking, these two aren't necessary during the build. Also, we are about to stop using `flatpak-session-helper` all together.

I'd also be careful about mentioning too many version numbers. They can go stale really quickly, and end up misleading people. I think the Podman version is the most important. It needs to be present at run-time on the host OS, and can have far-reaching consequences on the rest of the OS (think of OSes with long-term support).